plugin/modules.js

```diff
@@ -116,7 +116,7 @@
       var path = parentFile ? resolvePath(dirName(parentFile), word) : baseName(word)
       for (var prop in this.modules) {
         if (prop != parentFile && filter(path, prop, query)) {
-          if (/\.js$/.test(prop)) prop = prop.slice(0, prop.length - 3)
+          if (/\.jsx?$/.test(prop)) prop = prop.slice(0, prop.length - 3)
```

Comment:
There must be prop.length - 4 because length(".jsx") = 4.

Reply:
I found replace to work well here:
e.g.
prop = prop.replace(/\.jsx?/, '')